fe1-web/model/network/method/message/data/election/CreateElection.ts

```diff
@@ -0,0 +1,94 @@
+import {
+  Hash, Timestamp, Lao,
+} from 'model/objects';
+import { OpenedLaoStore } from 'store';
+import { ProtocolError } from 'model/network/ProtocolError';
+import { validateDataObject } from 'model/network/validation';
+import { ActionType, MessageData, ObjectType } from '../MessageData';
+import { checkTimestampStaleness } from '../Checker';
+
+export class CreateElection implements MessageData {
+  public readonly object: ObjectType = ObjectType.ELECTION;
+
+  public readonly action: ActionType = ActionType.SETUP;
+
+  public readonly id: Hash;
+
+  public readonly name: string;
+
+  public readonly version: string;
+
+  public readonly created_at: Timestamp;
+
+  public readonly start_time: Timestamp;
+
+  public readonly end_time: Timestamp;
+
+  public readonly questions: {
+    id: string,
+    question: string,
+    voting_method: string,
+    ballot_options: string[],
+    write_in: boolean,
+  }[];
```

Comment:
you might want to define a sub-type for this.

Reply:
Subtype is not defined as an interface in the model/objects/Election.ts file